### PR TITLE
Tx detail mobile

### LINKED ISSUE #596: Implement Transaction Details screen
https://invis.io/HJNS9ONZ8VA
https://invis.io/RGNS9RZ6YEK

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #596.

### CODE REVIEW

File: src/view/transaction-detail-mobile.js [hunk 1]
```diff
@@ -62,45 +53,44 @@ const TransactionDetailView = ({ store, nav }) => (
         <Title title="Transaction Detail" />
         {store.selectedTransaction.type === 'bitcoin' ? (
           <View style={[styles.icon, styles.btc]}>
-            <BitcoinIcon height={25} width={20} />
+            <BitcoinIcon height={135 * 0.19} width={170 * 0.19} />
```
Comment (valentinewallace) [inline src/view/transaction-detail-mobile.js]: How do you come to these formulas? I'm fine to do these changes myself so you don't have to refactor them in, but I'm not sure how you arrive at them :)

File: src/asset/img/orange-gradient-bg-mobile.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Svg, { Defs, LinearGradient, Stop, Path } from '../../component/svg';
+
+const SvgOrangeGradientBgMobile = props => (
```
Comment (valentinewallace) [inline src/asset/img/orange-gradient-bg-mobile.js]: Oo, one day i'll have to learn the art of the svg

**Comment by valentinewallace** [comment]
Looks great! Merging.

File: src/view/transaction-detail-mobile.js (hunk 1 above)
Comment (tanx) [reply src/view/transaction-detail-mobile.js]: I take the base height/width from the svg and kinda just play around with a scaling factor that looks good :D

**Pull request merged**